src/components/shared/LrSvgIcon/LrSvgIcon.vue

```diff
@@ -15,7 +15,8 @@
 * please email opensource@seagate.com.
 -->
 <template>
-  <div class="icon-container action-btn" v-if="icon">
+  <div class="icon-container" v-if="icon"
+    :class="[hoverIcon ? 'action-btn':'']">
```

Comment:
Instead of handling with class names, you can change the img src based onMouseOver.

Reply:
css hover will have both the events handled on mouseover , onmouseleave.

Reply:
There is one more way to handle it using CSS variables in which you can use CSS hover pseudo-class.

Reply:
we will be going with css hover